src/lib/hooks.js

```diff
@@ -69,6 +69,26 @@ const api = {
 		updateIframeGuestData({ department });
 	},
 
+	async setBusinessUnit(newBusinessUnit) {
+		if (!newBusinessUnit || !newBusinessUnit.trim().length) {
+			throw new Error('Error! Invalid business ids');
```

Comment:
Instead of throwing an error, can we just log the error to the console? Because a bad config here will disrupt how end users see the widget 😬 

(Not sure if that's inteded tho)

Reply:
I don't think this is required since the end-user wouldn't be affected if the business unit is not correct. It will just fall back to show all the departments instead of applying the BU filter. 
But let me know if you still think we should remove it.